### PR TITLE
EJDOTNETCORE-4254 Updated the README.md file

### PR DESCRIPTION
**Pull request by jeyalakshmit**
Hi sir,

I have updated the Readme.md file.
Kindly review and let me know your feedback.

Regards,
Jeyalakshmi T

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,37 @@
-# pdf-redaction-annotation-csharp
-This repository contains various examples to use the PDF redaction annotation
+# PDFRedactionAnnotation in C#
```
Comment (ghost) [inline README.md]: Redaction Annotation using C#

File: README.md [hunk 2]
```diff
@@ -1,2 +1,37 @@
...
+The [Syncfusion .NET PDF library](https://www.syncfusion.com/document-processing/pdf-framework/net) allows removing the sensitive text and images from the PDF document using its PDF Redaction feature. This repository contains various examples to use the PDF redaction annotation.
```
Comment (ghost) [inline README.md]: The [Syncfusion .NET PDF library]() provides an easy way to permanently remove confidential or sensitive information from PDF documents. This repository contains examples of redaction feature to safely, permanently remove confidential information from a PDF document.

File: README.md [hunk 3]
```diff
@@ -1,2 +1,37 @@
...
+
+Sample name | Description
+--- | ---
+[ContentRemoval](https://github.com/SyncfusionExamples/pdf-redaction-annotation-csharp/blob/main/.NET%20Framework/ContentRemoval) | Remove the marked content from PDF document.
```
Comment (ghost) [inline README.md]: Add the proper sample name and description for all the samples (as like in the blog)

File: README.md (hunk 1 above)
Comment (jeyalakshmit) [reply README.md]: Changed the title

File: README.md (hunk 2 above)
Comment (jeyalakshmit) [reply README.md]: changed the description.

File: README.md (hunk 3 above)
Comment (jeyalakshmit) [reply README.md]: changed the sample name and description

**Review by praveen-hari — APPROVED** [review]

**Pull request merged**